cmd/print.go

```diff
@@ -164,15 +189,24 @@ func newPrintCommand() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "print",
 		Short: "Prints HTTPRoutes and Gateways generated from Ingress resources",
-		RunE:  pr.PrintGatewaysAndHTTPRoutes,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if pr.allResources && pr.inputFile == "" {
+				return errors.New("--all-resources flag can be set only when --input-file is set")
+			}
+			return nil
+		},
```

Comment:
consider [cmd.MarkFlagsRequiredTogether](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsRequiredTogether) instead of PreRunE

Reply:
I initially considered using `cmd.MarkFlagsRequiredTogether`, but it turned out it cannot be used here, as `--input-file` can be used without `--all-resource`, while the other way around should be forbidden. As far as I get from `cmd.MarkFlagsRequiredTogether`, both flags are marked as required together, and is not possible to express the above.

Reply:
fair, thanks